src/api/db/structure.sql

```diff
@@ -1140,6 +1140,7 @@ CREATE TABLE `staging_request_exclusions` (
   `description` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `created_at` datetime NOT NULL,
   `updated_at` datetime NOT NULL,
+  `number` int(11) DEFAULT NULL,
```

Comment:
Why do we need to add the number now?

Reply:
@ChrisBr, we followed @coolo's [suggestion](https://github.com/openSUSE/open-build-service/pull/6201#discussion_r233365761). It was easier to add the 'number' column.